api/v1alpha1/obcluster_webhook.go

```diff
@@ -147,6 +147,14 @@ func (r *OBCluster) ValidateUpdate(old runtime.Object) (admission.Warnings, erro
 	} else if oldMode != oceanbaseconst.ModeStandalone && (oldCluster.Spec.OBServerTemplate.Resource.Cpu != r.Spec.OBServerTemplate.Resource.Cpu || oldCluster.Spec.OBServerTemplate.Resource.Memory != r.Spec.OBServerTemplate.Resource.Memory) {
 		return nil, errors.New("forbid to modify cpu or memory quota of non-standalone cluster")
 	}
+	if r.Spec.BackupVolume == nil && oldCluster.Spec.BackupVolume != nil {
+		return nil, errors.New("forbid to remove backup volume")
+	}
+	if r.Spec.BackupVolume != nil && oldCluster.Spec.BackupVolume == nil &&
+		mode != oceanbaseconst.ModeStandalone &&
```

Comment:
Is it possible to mount in normal mode, I think it only need to modify the container, the pod still remains

Reply:
It seems impossible to mount volume without recreating the pod after research, but we can still keep the ip using certain cni, which should be supported

Reply:
No, it's impossible to mount a new volume without modifying the pod. The `volumeMounts` field is in container but the declarations of `volumes` are in the spec of pod.

And by the way, if the service mode is available I don't think it is a good choice to deploy a "normal" mode cluster. :)

Reply:
OK, I'll check if the CNI is adopted.